### DISCUSSION TITLE
Can't start crazyflie after flashing ArduPilot firmware

### DISCUSSION THREAD

**Discussion by tchlandry** [bitcraze/discussions#1599] [Q&A]
Hi everyone,

I’ve encountered an issue with my Crazyflie 2.1. After flashing the ArduPilot firmware, the drone stays stuck with a solid blue LED. When I try to enter bootloader mode, the blue LED flashes rapidly, but the second LED does not light up. I’m unable to restore the original Crazyflie firmware using the client VM.

Any help on how to recover the Crazyflie or force the firmware reflash would be greatly appreciated!

Thanks!

**Comment by gemenerik** [comment]
Hi Ryland, please follow [this guide](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/recovery-mode/) to recover the firmware.

**Reply by tchlandry** [reply]
Hi Gemenerik

Thank you for your response.

Find the  solution by re-installe the bootloader with STM32CubeProgrammer and https://github.com/bitcraze/crazyflie2-stm-bootloader/releases

Do you know how Crazyflie communicate with QGroundControl after flashing ardupilot's firmware ?

The communication is good when I plug usb cable but wireless is not possible. Il tried to make a mavlink-bridge with ESP32 module on UART's port, it seems like this port is not active.

Do you have an idea ?

Thank you

**Reply by Bananazooka** [reply]
Hi, 
For ArduPilot communication, it's best to turn to them for questions as we're less familiar with that ecosystem.
